spec/lib/catalog/name_adjust_spec.rb

```diff
@@ -0,0 +1,17 @@
+describe Catalog::NameAdjust do
+  let(:item_name) { "MyItem" }
+  let(:multiple_names) { [item_name.to_s, "Copy of #{item_name}", "Copy (1) of #{item_name}"] }
+  let(:only_one_name) { [item_name.to_s, "anotheritem"] }
+
+  context "when passing in multiple Copy Names" do
```

Comment:
Small nitpick, there should be a surrounding `describe ".create_copy_name" do`, move the `let` on line 2 into that describe, and also move the `lets` on line 3/4 inside their respective contexts.